app/src/pages/common/membersPage/userNameCell/userNameCell.jsx

```diff
@@ -64,7 +65,7 @@ export const UserNameCell = ({ user, badges }) => {
 
 UserNameCell.propTypes = {
   user: PropTypes.shape({
-    id: PropTypes.string.isRequired,
+    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
```

Comment:
I'm guessing the `id` should always be a number according to the records processing in DB.

![image](https://github.com/user-attachments/assets/d706022f-7a54-45d2-9649-7f1e9ab67e26)
![image](https://github.com/user-attachments/assets/2f121b54-b8ff-4c4e-b1af-b78245174191)

So, it would be better to change the expected type to `number` and remove redundant conversions where applicable.